src/base/bittorrent/session.cpp

```diff
@@ -3762,7 +3762,7 @@ void Session::readAlerts()
         handleAlert(a);
 }
 
-void Session::handleAlert(libt::alert *a)
+void Session::handleAlert(const libt::alert *a)
```

Comment:
Such changes correspond to your second commit, not the first.

Reply:
Done.